internal/resources/grafana/resource_alerting_rule_group_test.go

```diff
@@ -384,6 +378,84 @@ resource "grafana_rule_group" "second" {
 	})
 }
 
+func TestAccAlertRule_ruleNameConflict(t *testing.T) {
+	testutils.CheckOSSTestsEnabled(t, ">=9.1.0")
+
+	name := acctest.RandString(10)
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProtoV5ProviderFactories: testutils.ProtoV5ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+resource "grafana_organization" "test" {
+	name = "%[1]s"
+}
+
+resource "grafana_folder" "first" {
+	org_id = grafana_organization.test.id
+	title = "%[1]s-first"
+}
+
+resource "grafana_rule_group" "first" {
+	org_id = grafana_organization.test.id
+	name             = "alert rule group"
+	folder_uid       = grafana_folder.first.uid
+	interval_seconds = 60
+	rule {
+		name           = "My Alert Rule"
+		for            = "2m"
+		condition      = "B"
+		no_data_state  = "NoData"
+		exec_err_state = "Alerting"
+		is_paused = false
+		data {
+			ref_id     = "A"
+			query_type = ""
+			relative_time_range {
+				from = 600
+				to   = 0
+			}
+			datasource_uid = "PD8C576611E62080A"
+			model = jsonencode({
+				hide          = false
+				intervalMs    = 1000
+				maxDataPoints = 43200
+				refId         = "A"
+			})
+		}
+	}
+	rule {
+		name           = "My Alert Rule"
+		for            = "2m"
+		condition      = "B"
+		no_data_state  = "NoData"
+		exec_err_state = "Alerting"
+		is_paused = false
+		data {
+			ref_id     = "A"
+			query_type = ""
+			relative_time_range {
+				from = 600
+				to   = 0
+			}
+			datasource_uid = "PD8C576611E62080A"
+			model = jsonencode({
+				hide          = false
+				intervalMs    = 1000
+				maxDataPoints = 43200
+				refId         = "A"
+			})
+		}
+	}
+}
+				`, name),
+				ExpectError: regexp.MustCompile(`rule with name "My Alert Rule" is defined more than oncendnndndndn`),
```

Comment:
That's probably wrong

Reply:
I fixed the bug in the unit test, but some 'Get Secrets' in jobs cannot be retrieved. How should I handle this?

Reply:
It's fine. These are just enterprise and cloud tests. All the tests that you added in this PR have passed.